db-commons/src/main/scala/org/bitcoins/db/CRUDAutoInc.scala

```diff
@@ -23,6 +24,12 @@ abstract class CRUDAutoInc[T <: DbRowAutoInc[T]](implicit
     safeDatabase.runVec(actions)
   }
 
+  override def updateAll(ts: Vector[T]): Future[Vector[T]] = {
+    FutureUtil.foldLeftAsync(Vector.empty[T], ts) { (accum, t) =>
```

Comment:
Can you link to the issue that you have filed here so there is some context given